### ISSUE TITLE
Rename dialog EditText is truncated on small screen

### ISSUE THREAD

**Issue by yura04** [zhanghai/MaterialFiles#214]
When I'm creating or renaming file or folder and opening keyboard it is hard to see what I'm renaming because letters are half-covered by interface.
I'm using fdroid build
![Screenshot_20191221-095523_Material_Files](https://user-images.githubusercontent.com/46564852/71305256-b7fe6780-23d9-11ea-905d-3375fcee9e09.png)

**Comment by afmbsr** [comment]
You must refer to the standard size for both the display and the keyboard

**Comment by yura04** [comment]
I had standard size of display and keyboard. But after your comment I have found the solution. I'm using AnySoftKeyboard and on my Redmi4x it was too large for this file manager.
So I've opened theme settings and changed " Keys height factor in portrait mode" from 1.0 to 0.95. And it works. Thanks.
But l think this bug will be problem for people with AnySoftKeyboard (and (?) small screen).

Anyway it's the best FOSS file manager i've seen and I will recommend it for my friends ;-)
